lib/govuk_schemas/version.rb

```diff
@@ -1,4 +1,4 @@
 module GovukSchemas
   # @private
-  VERSION = "2.3.0".freeze
+  VERSION = "3.0.0".freeze
```

Comment:
Can you do the version bump in a separate PR?

https://docs.publishing.service.gov.uk/manual/publishing-a-ruby-gem.html#versioning

Reply:
Done.. and got the tests to pass again. :)